### ISSUE TITLE
converters.py - allow indexing KeywordIndex

### ISSUE THREAD

**Issue by mtrebron** [collective/collective.dexteritytextindexer#20]
Keyword indices are currently not being picked up by the fallback converter. This could be achieved along the lines of the IntFieldConverter, e.g.:

```
#configure.zcml

    <adapter
        factory="collective.dexteritytextindexer.converters.TupleFieldConverter"
        />
```

```
#converters.py

from zope.schema.interfaces import ITuple
class TupleFieldConverter(DefaultDexterityTextIndexFieldConverter):
    """Converts the data of a Tuple field"""

    implements(interfaces.IDexterityTextIndexFieldConverter)
    adapts(IDexterityContent, ITuple, IWidget)

    def convert(self):
        """return the adapted field value"""
        storage = self.field.interface(self.context)
        value = self.field.get(storage)
        #LOGGER.info('Exporting field type:  %s' % self.field)
        #LOGGER.info('Converted %s field to: %s' % (self.widget.label.encode('utf-8'), str(value)))
        return str(value)
```

This would allow values entered through an AjaxSelectFieldWidget to become instantly searchable. 

```
<field name="my_keyword_field" type="zope.schema.Tuple" indexer:searchable="true" lingua:independent="true" form:widget="plone.app.z3cform.widget.AjaxSelectFieldWidget">
      <description>Type some keywords here</description>
      <required>False</required>
      <title>My Keywords</title>
      <value_type type="zope.schema.Choice">
        <vocabulary>plone.app.vocabularies.Keywords</vocabulary>
      </value_type>
    </field>  
```

2016-11-23 14:17:23 INFO collective.dexteritytextindexer Exporting field type:  <zope.schema._field.Tuple object at 0x7f7077d85bd0>
2016-11-23 14:17:23 INFO collective.dexteritytextindexer Converted My Keywords field to: (u'ham', u'spam', u'eggs')

**Comment by gforcada** [comment]
Hi @mtrebron thanks for reporting the issue!

Would you mind converting that to a pull request? If the code is already working for you it shouldn't be that much effort on doing that.

Although not needed for collective repositories, but it would be nice if you would like to sign the Plone Contributors Agreement: https://plone.org/foundation/contributors-agreement

**Comment by mtrebron** [comment]
Hi @gforcada 
I'll find time to convert it in the next few days. Thanks for pointing me to contributor agreement, will check it out.

**Comment by gforcada** [comment]
no problem, let us know if you need help/advice

**Comment by jone** [comment]
I believe this is fixed in `2.2.0` by #27
